spec.html

```diff
@@ -38888,7 +38894,10 @@ <h1>%AsyncFromSyncIteratorPrototype%.throw ( _value_ )</h1>
             1. If _throw_ is *undefined*, then
               1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; _value_ &raquo;).
               1. Return _promiseCapability_.[[Promise]].
-            1. Let _result_ be Call(_throw_, _syncIterator_, &laquo; _value_ &raquo;).
+            1. If _value_ is present, then
```

Comment:
`throw` method seems to have only one call site that always passes an argument: [last production, step 7.b.ii.1](https://tc39.es/ecma262/#sec-generator-function-definitions-runtime-semantics-evaluation). Is it possible for `value` to absent here?

Reply:
Perhaps not, but it seems better to be consistent, to prevent a footgun if in the future, a callsite for it is added?

Reply:
Absolutely, especially if runtimes use a shared implementation for the `AsyncFromSyncIterator` methods (like [SpiderMonkey does](https://github.com/mozilla/gecko-dev/commit/d6a7fbec08c618f68965a06e954886391e27d111)).